platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -1,10 +1,13 @@
 {#-
-    Copyright (c) 2020-2023 NVIDIA CORPORATION & AFFILIATES.
+    Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
```

Comment:
It should be 2020-2024 as the file is not new

Reply:
Will update